src/test/java/com/google/api/codegen/protoannotations/testdata/java_library_no_gapic_config.baseline

```diff
@@ -3953,7 +4546,198 @@ public class LibraryServiceStubSettings extends StubSettings<LibraryServiceStubS
     privateGetBookSettings = settingsBuilder.privateGetBookSettings().build();
   }
 
+  private static final PagedListDescriptor<ListShelvesRequest, ListShelvesResponse, Shelf> LIST_SHELVES_PAGE_STR_DESC =
```

Comment:
These PagedListDescriptors are also present in java_library.baseline: https://github.com/googleapis/gapic-generator/blob/eea42da690a5cec5660a76c540a45465323128af/src/test/java/com/google/api/codegen/gapic/testdata/java/java_library.baseline#L6459